src/core/MOM_forcing_type.F90

```diff
@@ -2910,6 +2918,10 @@ subroutine forcing_diagnostics(fluxes_in, sfc_state, G_in, US, time_end, diag, h
     if ((handles%id_ustar_ice_cover > 0) .and. associated(fluxes%ustar_shelf)) &
       call post_data(handles%id_ustar_ice_cover, fluxes%ustar_shelf, diag)
 
+    if ((handles%id_shelf_sfc_mass_flux > 0) &
```

Comment:
There does not appear to be a register_diag_field call anywhere for handles%id_shelf_sfc_mass_flux. Is this deliberate, or is this being handled in some other way?